tools/genesis-data/schemas.mjs

```diff
@@ -0,0 +1,17 @@
+// This small nodejs script will pull the schemas from mainnet and then output them in Genesis Schema format
+
+import { getSchemas } from "./getSchema.mjs";
+
+const SOURCE_URL = "wss://1.rpc.frequency.xyz";
+
+async function main() {
+  try {
+    await getSchemas(SOURCE_URL);
+    process.exit(0);
+  } catch (error) {
+    console.error("Error:", error);
+    process.exit(1);
+  }
+}
+
+main();
```

Comment:
Best practice is probably to have at least a `.catch()` or `.then()` on this. Usually I just move the `try...catch` from the main function into a `.catch(...)` on the `main()` invocation